bundles/scripting/sightly/testing-content/src/main/resources/SLING-INF/apps/sightlyperf/test/sly-js-sync.html

```diff
@@ -18,6 +18,7 @@
     ${test.tag != null ? test.startTag : '' @ context = "unsafe"}
     ${test.text @ context = "text"}
     ${test.tag != null ? test.endTag : '' @ context = "unsafe"}
+    ${'yyyy-MM-dd HH:mm:ss.SSS' @ format = test.date}
```

Comment:
The functionality tests are part of the HTL TCK (https://github.com/Adobe-Marketing-Cloud/htl-tck/pull/20/files#diff-05ef2ebd0db1e547f6e90524ab379328R39). The tests here are just for performance evaluation and I think this covers the worst scenario.